### DISCUSSION TITLE
Declaring a "Performing Artist" class in Artsdata

### DISCUSSION THREAD

**Discussion by fjjulien** [culturecreates/artsdata-data-model#131] [Data Model]
This discussion is related to https://github.com/culturecreates/artsdata-data-model/discussions/130

### Background
As a [recent CAPACOA report](https://capacoa.ca/en/2024/07/a-tipping-point-towards-open-data/) found, many industry platforms have a concept designating an "artist" or "performer" agent that can be either a `schema:Person` or `schema:Organization` (or subclass thereof).

### Specific use case
The Arts Touring Connector (ATC) is a platform designed to facilitate the planning of show tours. This process, known as "block programming", is carried out by presenting organizations and the networks that support them. Two key classes closely associated with the Tour concept in the ATC data model are: `Artist` and `Show`. As a first step in the ATC-Artsdata integration, ATC would like their users to be able to autocomplete Artist names against Artsdata entities.

### Problem
Artsdata does not have an equivalent class to the ATC Artist class. Therefore ATC developers must reconcile Artists twice: once against Person type entities, and once against Organization type entities. The same problem may also be experienced as I Want To Showcase starts reconciling "Performer" entities with Artsdata.

### Solution
As an immediate solution, discussion https://github.com/culturecreates/artsdata-data-model/discussions/130 is proposing to to integrate an "Agent" class into the Artsdata data model.

As longer-term solution, we may want to declare a narrower class (i.e. a subclass of "Agent") that would define a range for the [schema:performer](https://schema.org/performer) in Artsdata. In other words, the class would designate a person, group or organization that one could expect to see as the main performer in an event.

This class could automatically be assigned when certain conditions are met. For example:

- An entity has the "Artist" class in the Arts Touring Connector;
- An entity had the "Performer" class in I Want To Showcase;
- An entity has the "Artiste" class in Scènes francophones;
- An entity is stated a performer of an event in event structure data and the structured data includes a sameAs pointing to a URI clearly identifying this entity.

**Comment by saumier** [comment]
@fjjulien Would you like me to proceed and declare a new Artsdata Class for Artist? 

A minimal definition for http://kg.artsdata.ca/ontology/Artist
- rdf:type rdfs:Class
- rdfs:label Artist@en, Artiste@fr
- rdfs:comment   "a person, group or organization that one could expect to see as the main performer in an event"@en
- rdfs:subClassOf dbo:Agent

**Reply by fjjulien** [reply]
@saumier Your minimal definition is fine.

However, I am now hesitant about declaring this class.

- What would an ad:Artist call enable that dbo:Agent could not? 
  - Would we enable reconciliation against this class? If so, what if an artist has the dbo:Agent type, but not ad:Artist? Would it rank lower in the API response?
  - Would the class be made available to Footlight CRM users?
- How would we assign the ad:Artist class?
  - Would we use the rules that I suggested above?
  - Would we define additional rules?

I would like us to have more clarity on the use cases and the potential workflows before proceeding.

**Reply by saumier** [reply]
@fjjulien Agreed. I think we should advance slowly and gather more use cases.  I will take this out of the IWTS Work Order so we can give it more time.
